scripts/test-brand-preservation.js

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env node
+
+/**
+ * Brand Preservation Test Script
+ * 
+ * This script validates the fix for CSS specificity conflicts that occur when
+ * switching between light and dark themes with custom brand images.
+ * 
+ * Issue: https://github.com/storybook-community/storybook-dark-mode/issues/22
+ * 
+ * Usage: node scripts/test-brand-preservation.js
+ */
+
+console.log('🧪 Testing Brand Image Preservation Fix\n');
+
+// Mock the addons API to simulate Storybook's behavior
+const mockConfig = {
+  theme: {
+    brandImage: 'https://example.com/my-custom-logo.png',
+    brandTitle: 'My Custom App',
+    brandUrl: 'https://example.com',
+    brandTarget: '_blank',
+    // Other theme properties would be here too
+    base: 'light',
+    colorPrimary: '#FF4785',
+    colorSecondary: '#029CFD'
+  }
+};
+
+const mockAddons = {
+  getConfig: () => mockConfig
+};
+
+// Mock the themes
+const mockThemes = {
+  light: {
+    base: 'light',
+    colorPrimary: '#FF4785',
+    colorSecondary: '#029CFD',
+    // Note: no brand properties in default themes
+  },
+  dark: {
+    base: 'dark', 
+    colorPrimary: '#FF4785',
+    colorSecondary: '#029CFD',
+    // Note: no brand properties in default themes
+  }
+};
+
+// Simulate the OLD behavior (before our fix)
+function oldBehavior(mode) {
+  console.log(`❌ OLD: Switching to ${mode} theme...`);
+  const newTheme = mockThemes[mode]; // This loses brand customizations
+  console.log('   Result:', {
```

Comment:
You can avoid the lint error by using `console.info()`.

But I would love to see (maybe I can amend the PR with) making this part of the regular test harness so that it can be validated in CI.